packages/near-api-js/package.json

```
@@ -0,0 +1,80 @@
+{
+    "name": "@near-api-js/near-api-js",
```

```suggestion
    "name": "near-api-js",
```
This will change where are publishing to so let's keep this as `near-api-js`. We can scope all other (new) packages.